pkg/client/apiutil/restmapper.go

```diff
@@ -196,52 +211,64 @@ func (m *lazyRESTMapper) addKnownGroupAndReload(groupName string, versions ...st
 	}
 
 	// Update data in the cache.
+	m.mu.Lock()
 	m.knownGroups[groupName] = groupResources
```

Comment:
I assume it's not a problem if we have concurrent calls for the same groupName as they will end up with the same groupResources?